.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
disabled blank issues and added a `Anything else` template, in order to ensure that all new issues are auto-labeled with `needs-triage`